src/components/InQueueMessaging/InQueueCallbackComponent.js

```diff
@@ -85,11 +86,10 @@ class InQueueCallbackComponent extends React.Component {
       state: state
     })
       .then(data => {
-    	console.log(baseUrl);
         console.log('==== cbUiPlugin web service success ====');
       })
       .catch(error => {
-        console.log('cbUiPlugin web service error');
+        console.log('cbUiPlugin web service error' + error);
```

Comment:
I recommend `console.log('cbUiPlugin web service error', error);` instead - this way the `error` object will be logged and can be viewed like an object in chrome dev tool